modules/osd-aws-privatelink-about.adoc

```diff
@@ -5,3 +5,8 @@
 A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster to avoid all public ingress network traffic. Without a public subnet, it is not possible to configure an application router as public. Configuring private application routers is the only option.
 
 For more information, see link:https://aws.amazon.com/privatelink/[AWS PrivateLink] on the AWS website.
+
+[IMPORTANT]
+====
+You can only make a PrivateLink cluster at installation time. You cannot change a cluster to private after installation.
```

Comment:
I think this needs to be changed from "You cannot change a cluster to private after installation." to "You cannot change a cluster to PrivateLink after installation."